js/snippetPreview.js

```diff
@@ -1146,6 +1153,52 @@ SnippetPreview.prototype.setMetaDescription = function( metaDesc ) {
 	this.changedInput();
 };
 
+/**
+ * Creates elements with the purpose to calculate the sizes of elements and puts these elemenents to the body.
+ */
+SnippetPreview.prototype.createMeasurementElements = function() {
+	var titleElement = document.createElement( 'span' );
```

Comment:
Please change all quotes to double quotes.
The build currently fails because of this usage of single quotes.